Code/Tools/SceneAPI/SceneCore/Utilities/DebugOutput.cpp

```diff
@@ -251,6 +251,19 @@ namespace AZ::SceneAPI::Utilities
             productList.AddProduct(
                 (productName + ".xml"), AZ::Uuid::CreateName((productName + ".xml").c_str()), dbgSceneGraphXmlAssetType,
                 AZStd::nullopt, AZStd::nullopt);
+
+            // save out debug text for the Scene Manifest
+            AZStd::string productNameDebugManifest { debugSceneFile };
```

Comment:
Should this be behind a cvar so it can be turned as needed? 

Might be useful to cover how of these we may generate per run (ie one per level) and/or size information.

Reply:
it is, the "--debugOutput" command line arg determines if this should be outputted or not.
This is equivalent to outputting the Scene Graph debug file, so this continues the current method of debugging FBX files.